internal/commands/push/command.go

```diff
@@ -105,6 +127,12 @@ func (cmd *Command) Inputs() cli.InputResolver {
 
 // Handler is the command handler
 func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.Clients) error {
+	if cmd.inputs.IncludeNodeModules && cmd.inputs.IncludePackageJSON {
```

Comment:
same thing about putting this logic into `inputs.Resolve` and trimming down the error message so that it makes sense when printed alongside the "usage info"